test/policies/episode_policy_test.rb

```diff
@@ -57,5 +62,17 @@ def token(scopes, set_account_id=account_id)
       episode.published_at = nil
       refute EpisodePolicy.new(draft_token, episode).update?
     end
+
+    it 'allows deleting a draft episode' do
+      episode = create(:episode, published_at: nil, podcast: podcast)
+
+      assert EpisodePolicy.new(draft_token, episode).destroy?
+    end
+
+    it 'does not allow deleting a non-draft episode' do
+      episode = create(:episode, published_at: 25.years.ago, podcast: create(:podcast, prx_account_uri: "/api/v1/accounts/#{account_id}"))
```

Comment:
Why the create-podcast here, instead of using the build_stubbed'd one?

Reply:
Tests were giving me guff about build_stubbed touching the database